src/iris/api.py

```diff
@@ -1418,9 +1475,10 @@ def on_get(self, req, resp):
         where += gen_where_filter_clause(
             connection, plan_filters, plan_filter_types, req.params)
 
-        tag_subquery = gen_tag_where_subquery(connection, 'plan_id', 'plan_metadata_tag', '`plan`.`id`', req.params)
-        if tag_subquery != "":
-            where.append(tag_subquery)
+        tag_subqueries = gen_tag_wheres(connection, 'plan_id', 'plan_metadata_tag', '`plan`.`id`', req.params)
+        if tag_subqueries:
+            for tag_subquery in tag_subqueries:
+                where.append(tag_subquery)
```

Comment:
Can avoid some boilerplate by replacing the door loop with where.extend()

Reply:
changed